source/playgrounds.txt

```diff
@@ -30,12 +30,40 @@ files with the ``.mongodb`` extension as playgrounds.
 
 .. include:: /includes/admonitions/note-playgrounds.rst
 
-|vsce| splits your Playground window to display the results of your 
-Playground in |json| format in the right-side pane labeled ``Playground 
-Results.json``. If you disabled split-view, |vsce| displays the 
-Playground results in |json| format in a new tab to the right of 
-your Playground. You cannot edit the results in the results pane. 
-You can save the results to a file. 
+- If you enabled split-view, |vsce| splits your Playground window to
+  display the results of your Playground.
+
+  - If the results can be displayed in |json| format, |vsce| labels the
+    right-side pane **Playground Results.json**.
+
+  - If the results can't be displayed in |json| format, |vsce| labels
+    the right-side pane **Playground Result**.
+
+- If you disabled split-view, |vsce| displays the Playground results in
+  a new tab to the right of your Playground. These results may be in in
+  |json| format if possible.
+
+In the Playground results tab, you can:
+
+- Save the results to a file.
+- Edit the results in the results pane or tab if you made a
+  :doc:`findOne or find query </read-document-playground>` in the
+  Playground.
+
+  1. Edit the document.
+
+     - If the playground returned one |json| document, click
+       :guilabel:`Edit Document` at the top of that document.
+
+     - If the playground a list of |json| documents, click
+       :guilabel:`Edit Document` at the top of each |json| document.
```

Comment:
> If the playground a list of |json| documents, click `Edit Document` at the top of each |json| document.

This is also confusing.

If the playground returned a list of |json| documents, click `Edit Document` at the top of the |json| document you want to edit.